lib/msf/base/sessions/command_shell_options.rb

```diff
@@ -17,9 +17,11 @@ def initialize(info = {})
 
     register_advanced_options(
       [
-        OptString.new('InitialAutoRunScript', [false, "An initial script to run on session creation (before AutoRunScript)", '']),
-        OptString.new('AutoRunScript', [false, "A script to run automatically on session creation.", ''])
-      ], self.class)
+        OptString.new('InitialAutoRunScript', "An initial script to run on session creation (before AutoRunScript)"),
+        OptString.new('AutoRunScript', "A script to run automatically on session creation."),
+        OptString.new('CommandShellCleanupCommand', "A command to run before the session is closed")
```

Comment:
We could also leave this unregistered, and simply allow the payloads that need it to define it. This also brought up the question of the best way for a payload module to override the default value without having to entirely redefine the datastore option, which would be a nice feature.

Reply:
What about `DefaultOptions`?

Reply:
that's perfect, thanks

Reply:
For the record, we discussed this and think it should stay registered here so users can see that it's an option (and it'll be tab-completable). Module authors can set a default in `DefaultOptions`.